### ISSUE TITLE
Execute action after rofi window is closed

### ISSUE THREAD

**Issue by AurelWeinhold** [jluttine/rofi-power-menu#25]
Hi,

 is it possible to have the action performed after the rofi window is closed?
I have adapted the script to have a utility menu for my desktop with one of the utilities being to take a screenshot and do some processing on that. Of course I would like to take the screenshot without the rofi utility menu open.

Thank you for for the script!

**Comment by wesker-albert** [comment]
@AurelWeinhold did you ever discover a solution for this?

**Comment by AurelWeinhold** [comment]
> @AurelWeinhold did you ever discover a solution for this?

No, unfortunately not. I have moved to using a keyboard shortcut for screenshot-ing instead.
